node/src/Pos/Util/Monitor.hs

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs            #-}
+{-# LANGUAGE RankNTypes       #-}
+
+module Pos.Util.Monitor (
+
+      startMonitor
+    , stopMonitor
+
+    ) where
+
+import           Control.Concurrent           (killThread)
+import           Control.Monad.IO.Class
+import           Mockable.Class
+import qualified Mockable.Metrics             as Metrics
+import           Node
```

Comment:
Imports should be qualified or contain explicit import list, according to [style-guide](https://github.com/input-output-hk/cardano-sl-style-guides/blob/master/haskell-style-guide.md#imports).